packages/binding-http/test/http-server-test.ts

```diff
@@ -160,6 +161,7 @@ class HttpServerTest {
     await httpServer.start(null);
     
     let testThing = new ExposedThing(null);
+    testThing.title = "Test";
```

Comment:
Because otherwise, you end up with `slugify(undefined)` which will throw an exception. And also, according to https://www.w3.org/TR/wot-thing-description/#thing, the `title` field is mandatory.
Do you think there should be a different solution for that?

Reply:
Ah ok, there was a time where the title was not mandatory. So yes I think is fine as it is 👍 .